docs/components/SkipToContent.tsx

```diff
@@ -0,0 +1,44 @@
+import { css } from '@emotion/react';
+import { theme } from '@expo/styleguide';
+import React from 'react';
+
+const SKIP_TO_CONTENT_STYLES = css`
+  text-decoration: none;
+  position: absolute;
+  left: -100%;
+  top: -100%;
+  z-index: 9999;
+  border-radius: 4px;
+
+  background: ${theme.button.tertiary};
+  padding: 8px;
+
+  &:focus {
+    left: 16px;
+    top: 16px;
+  }
+`;
+
+export interface SkipToContentProps {
```

Comment:
I think we mostly prefer or use `type SkipToContentProps = {}` over interfaces. It's a minor thing